test/jsonValidationTest.js

```diff
@@ -5,120 +5,167 @@ const fs = require('fs');
 describe('Validating Cohorts...', () => {
     const cohortPath = './Cohorts';
 
-    it('Verifying .cohort files', function(done){
+    it('Verifying .cohort files', function (done) {
         let failedList = [];
         browseDirectory(cohortPath, (error, results) => {
             results.filter(file => file.substr(-7) === '.cohort')
-                .forEach(file => {                    
-                    validateJsonString(file)
+                .forEach(file => {
+                    validateJsonStringAndGetObject(file)
```

Comment:
I *think* you can go all "javascript" on this block, and just
```
.forEach(validateJsonStringAndGetObject);
```
if you're just passing the param directly to the method

Reply:
@gardnerjr I am not sure what you mean by that.. Could you elaborate more?

Reply:
i believe you can replace this block:

```.forEach(file => { 
validateJsonStringAndGetObject(file);
});
```

with `.forEach(validateJsonStringAndGetObject);`

Reply:
I will just leave as is since it is not significant and leave it for more readability. :-)